build/configs/common_download.sh

```diff
@@ -258,10 +258,13 @@ download_specific_partition()
 		echo "No corresponding binary for the partition ${parts[$partidx]}"
 		echo "Download $exe_name FAILED!"
 	else
-		board_download $TTYDEV ${offsets[$partidx]} ${exe_name} ${sizes[partidx]} ${parts[$partidx]}
+		board_download_separate $TTYDEV ${offsets[$partidx]} ${exe_name} ${sizes[partidx]} ${parts[$partidx]}
```

Comment:
Could you please check other chips also?

Reply:
for other chips we need to update the <board>_download.sh script. Realtek raise a new PR with updated linux tool and the required changes. I checked rtl8721csm/hello, it is working well for both erase and download for all  and individual binaries

Reply:
Plus also we need to update the board_metadata.txt for other boards 
Either we need to remove default_port or change it from ttyUSB1 to /dev/ttyUSB1